src/main/java/com/jcabi/github/mock/MkComments.java

```diff
@@ -146,15 +146,22 @@ public Comment post(
         this.storage.lock();
         final int number;
         try {
+            final String timestamp = new Github.Time().toString();
+            final String userrepo = this.repo.user() + "/" + this.repo.repo();
             number = 1 + this.storage.xml()
                 .nodes("//comment/number").size();
+            final String url = "https://api.jcabi-github.invalid/repos/"
+                + userrepo + "/issues/comments/" + number;
             this.storage.apply(
                 new Directives().xpath(this.xpath()).add("comment")
                     .add("number").set(Integer.toString(number)).up()
+                    .add("url").set(url).up()
```

Comment:
let's avoid using one-time variables. Please, inline url here as following:
```java
.add("url")
.set(
   String.format(
       "https://api.jcabi-github.invalid/repos/%s/%s/issues/comments/%d", 
        this.repo.user(),
        this.repo.repo(),
        number
   )
)
```

and remove ```userrepo``` and ```url``` variables